pom.xml

```
@@ -247,11 +236,36 @@
     </dependencies>
   </dependencyManagement>
 
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <scm>
+    <connection>scm:git:https://github.com/flyteorg/flytekit-java.git</connection>
+    <developerConnection>scm:git:https://github.com/flyteorg/flytekit-java.git
+    </developerConnection>
+    <url>https://github.com/flyteorg/flytekit-java</url>
+    <tag>HEAD</tag>
+  </scm>
+  <developers>
+    <developer>
+      <name>flytekit-java</name>
+      <email>dataex@spotify.com</email>
+      <organization>Flyte</organization>
+      <organizationUrl>https://flyte.org</organizationUrl>
+    </developer>
+  </developers>
   <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

```suggestion
      <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
```
According to instructions [here](https://central.sonatype.org/publish/publish-guide/) this should be the new url